db/models.py

```diff
@@ -1,4 +1,10 @@
 from django.db import models
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
+
+
+class User(AbstractUser):
+    pass
```

Comment:
Ensure that the `AUTH_USER_MODEL` setting in your Django settings is set to this custom `User` model to avoid conflicts with Django's default user model.